cms-publish-worker-webapp/src/main/java/se/simonsoft/cms/publish/worker/AwsStepfunctionPublishWorker.java

```diff
@@ -168,6 +173,17 @@ public void run() {
 								updateStatusReport("Retrieving", new Date(), "Ticket: " + publishTicket.toString());
 								
 								String exportPath = exportCompletedJob(publishTicket, options);
+								// Delete the temporary directory if needed
```

Comment:
Can we move this code into PublishJobService? Perhaps into getCompletedJob(..) btw isComplete and the actual retrieval of the result.

Reply:
of course.